test/runtests.jl

```diff
@@ -84,5 +84,17 @@ end
     end
 
 end
+    
+@testset "isordered" begin
+
+    @test isordered([1])
```

Comment:
Tests are failing because `isordered` isn't exported from DataAPI; I'd rather not export to be consistent with other DataAPI definitions, but I also think it's probably a fairly safe case if we really want to.

Reply:
Agreed. I will fix it - every time I think that the PR is simple (and I just edit it in the web browser) I mess up things. I should be more careful.